yarn-project/types/src/constants.ts

```diff
@@ -1 +1,2 @@
 export const INITIAL_L2_BLOCK_NUM = 1;
+export const BLOB_SIZE_IN_BYTES = 31 * 4096;
```

Comment:
![image](https://github.com/AztecProtocol/aztec-packages/assets/16536249/94f5af28-40b7-437b-b076-a7a8c1e26069)

Used for warning around deployment size. Brillig produced contracts larger than what could be deployed so broke tests etc.

A few deployments with public functions and all the blobs are gone if you are even able to deploy because of contract size. 

![image](https://github.com/AztecProtocol/aztec-packages/assets/16536249/f6bcdd79-83fa-4804-8f01-78b667ad4209)